specs/eventing/control-plane.md

```diff
@@ -0,0 +1,1007 @@
+  </tr>
+  <tr>
+    <td><code>backoffPolicy</code></td>
+    <td>enum<br/>("linear", "exponential")</td>
+    <td>Retry timing scaling policy. Linear policy uses the same <code>backoffDelay</code> for each attempt; Exponential policy uses 2^N multiples of <code>backoffDelay</code></td>
+    <td>RECOMMENDED</td>
+  </tr>
+</table>
+
+### KReference
+
+KReference is a lightweight version of kubernetes
+[`v1/ObjectReference`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#objectreference-v1-core)
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>apiVersion</code></td>
+    <td>string</td>
+    <td>ApiVersion of the target reference.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>kind</code></td>
+    <td>string</td>
+    <td>Kind of the target reference.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>namespace</code></td>
+    <td>string</td>
+    <td>Namespace of the target resource.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>address</code></td>
+    <td>string</td>
+    <td>Address used to deliver events to the Broker.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+### PhysicalSubscriptionStatus
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>subscriberUri</code></td>
+    <td>URL (string)</td>
+    <td>Resolved address of the <code>spec.subscriber</code>.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>replyUri</code></td>
+    <td>URL (string)</td>
+    <td>Resolved address of the <code>spec.reply</code>.</td>
+    <td>REQUIRED</td>
+  </tr>
+  <tr>
+    <td><code>deadLetterSinkUri</code></td>
+    <td>URL (string)</td>
+    <td>Resolved address of the <code>spec.delivery.deadLetterSink</code>.</td>
+    <td>REQUIRED</td>
+  </tr>
+</table>
+
+### TriggerFilter
+
+<table>
+  <tr>
+    <td><strong>Field Name</strong></td>
+    <td><strong>Field Type</strong></td>
+    <td><strong>Description</strong></td>
+    <td><strong>Schema Requirement</strong></td>
+  </tr>
+  <tr>
+    <td><code>attributes</code></td>
+    <td>map[string]string</td>
+    <td>Event filter using exact match on event context attributes. Each key in the map MUST be compared with the equivalent key in the event context. All keys MUST match (as described below) the event attributes for the event to be selected by the Trigger.
+    <br>
+    For each key specified in the filter, an attribute with that name MUST be present in the event to match. If the value corresponding to the key is non-empty, the value MUST be an exact match to attribute value in the event; an empty string MUST match all attribute values.</td>
+    <td>REQUIRED</td>F
```

Comment:
The "F" at the end shows up before the table for me:
<img width="417" alt="image" src="https://user-images.githubusercontent.com/1944671/126161064-7d057eac-9391-42cd-84b1-88c58006fcb1.png">